content/blog/version-field/contents.lr

```diff
@@ -0,0 +1,26 @@
+title: Version field launched!
+---
+author: aioannidis
+---
+body:
+Today we are introducing a version tag field in our metadata, in order for you to be able to explicitly specify the version of your software, dataset, or any type of upload over which a versioning scheme could be applied. This change is also part of our efforts to make software a first-class citizen in the world of scientific research.
```

Comment:
Last blogpost started similarly, maybe change it to something else

"The version field has been added to our metadata schema."
"As of today, "version" has become a new field in our metadata - you can now explicitly specify the version of your record."